OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessage.java

```diff
@@ -34,24 +43,42 @@
     @NonNull
     public ArrayList<ArrayList<OSTrigger>> triggers;
 
+    //Last IAM display time in seconds
+    private long lastDisplayTime = -1;
+    //Current quantity of displays
+    private int displaysQuantity = 0;
+    //Quantity of displays limit
+    private int displaysLimit = Integer.MAX_VALUE;
+    //Delay between displays in seconds
+    private long delay = 0;
     private double displayDuration;
 
+    private boolean reDisplayEnabled = false;
```

Comment:
`reDisplayEnabled` should be `redisplayEnabled`

Redisplay is one word, so no capitol D